### PR TITLE
chore: add mainnet deployments and update some config

### PR DESCRIPTION
**Pull request by Pangssu**
# Description

- Add mainnet contract deployments.
- Add mainnet informations to kroma-node chaincfg, sdk.
- Set kroma-node default bootnodes.

### CODE REVIEW

File: components/node/chaincfg/chains.go [hunk 1]
```diff
@@ -10,6 +10,36 @@ import (
...
+	Genesis: rollup.Genesis{
+		L1: eth.BlockID{
+			Hash:   common.HexToHash("0xe459c500b760ed52a1ad799bf578b257af2c76f6ebe061a4c62627e9c605bced"),
+			Number: 18067255,
+		},
+		L2: eth.BlockID{
+			Hash:   common.HexToHash("0xeab1dbcbd854942126643609f6b457e391b169c819b7e5d5042389ccf6012cbf"),
+			Number: 0,
+		},
+		L2Time: 1688709132,
```
Comment (0xHansLee) [inline components/node/chaincfg/chains.go:23]: maybe?
```suggestion
		L2Time: 1693880387,
```

**Review by seolaoh — APPROVED** [review]
LGTM

**Review by 0xHansLee — APPROVED** [review]
LGTM

**Review by kangsorang — APPROVED** [review]
lgtm

**Pull request merged**